cmds/webboot/webboot.go

```diff
@@ -8,20 +8,40 @@ import (
 	"regexp"
 
 	ui "github.com/gizak/termui/v3"
+	"github.com/u-root/webboot/pkg/bootiso"
 	"github.com/u-root/webboot/pkg/menu"
 )
 
 var (
 	v       = flag.Bool("verbose", false, "Verbose output")
 	verbose = func(string, ...interface{}) {}
 	dir     = flag.String("dir", "", "Path of cached directory")
-	network = flag.Bool("network", true, "Should  or not set up network")
+	network = flag.Bool("network", true, "Should or not set up network")
+	boot    = flag.Bool("boot", true, "Should or not boot the iso. May trun it off for test.")
```

Comment:
Usually we name such flags dry-run, so if dry-run is true then we wouldn't boot the iso